tests/lua/run.sh

```
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Runs the standalone (engine-free) Lua tests with a plain Lua 5.5 interpreter.
+# Usage: bash tests/lua/run.sh [path-to-lua]
+set -u
+
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)